### PR TITLE
update olm bindata

### LINKED ISSUE #5680: SDK does not officially support the latest OLM version
SDK has a list of officially supported OLM versions which can be used with `operator-sdk olm` commands (https://sdk.operatorframework.io/docs/overview/#olm-version-compatibility). The latest OLM's release is 0.20.0 (https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.20.0), but we still support 0.18.2.

Solution:
1. Verify if everything in SDK works with OLM's latest version.
2. Bump olm-bindata. The steps are here: https://sdk.operatorframework.io/docs/contribution-guidelines/releasing/#updating-olm-bindata
3. Update docs.

### PR DESCRIPTION
**Pull request by everettraven**
**Description of the change:**
operator-sdk was behind on a few OLM versions. updated the supported versions and bindata.

**Motivation for the change:**
resolves #5680

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [x] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by everettraven** [comment]
Does this need a changelog update?

**Comment by everettraven** [comment]
Also, this updates to use the 3 latest minor version releases (0.18.3, 0.19.1, 0.20.0). Do we need to instead update it to use the 3 latest releases (0.19.0, 0.19.1, 0.20.0)?

**Review by varshaprasad96 — COMMENTED** [review]
@everettraven Thanks for the PR! Yes, let's modify it to support the latest 3 versions (0.19.0, 0.19.1, 0.20.0). Also, you may have to update the available olm versions here: https://github.com/operator-framework/operator-sdk/blob/04fb844950722eef4e70fe7a1bfc656c712a7a85/internal/bindata/olm/versions.go#L17 for the e2e integration test suite to pass.

**Comment by varshaprasad96** [comment]
Yes, let's add changelog too.

File: website/content/en/docs/overview/_index.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ By project type (replace `${IMAGE_VERSION}` with base image version in your proj
 
 Operator SDK officially supports the latest 3 versions of OLM present at the time of a given Operator SDK release. These versions of OLM manifests are packaged with the SDK binary in the form of `bindata` to support low-latency installations of OLM with [`operator-sdk olm install`][olm-install-cmd]. Any other version installed with this command may work but is not tested nor officially supported.
 
-Currently, the officially supported OLM Versions are: 0.16.1, 0.17.0 and 0.18.2.
+Currently, the officially supported OLM Versions are: 0.19.0, 0.19.1, and 0.20.0
```
Comment (varshaprasad96) [inline website/content/en/docs/overview/_index.md:95]: Just nit:
```suggestion
Currently, the officially supported OLM Versions are: 0.19.0, 0.19.1, and 0.20.0.
```

**Review by varshaprasad96 — APPROVED** [review]
/lgtm

File: Makefile [hunk 2]
```diff
@@ -44,7 +44,7 @@ generate: build # Generate CLI docs and samples
 	go generate ./...
 
 .PHONY: bindata
-OLM_VERSIONS = 0.16.1 0.17.0 0.18.2
+OLM_VERSIONS = 0.19.0 0.19.1 0.20.0
```
Comment (jmrodri) [inline Makefile:47]: It seems weird to have both `0.19.0` and `0.19.1`. I would have expected `0.18.3`, `0.19.1`, & `0.20.0`.

**Review by jmrodri — COMMENTED** [review]
I don't really see the value of someone installing an earlier version of OLM that already has a point release fix out. i.e. why would I want `0.19.0` instead of `0.19.1`.

File: Makefile (hunk 2 above)
Comment (everettraven) [reply Makefile]: updated in https://github.com/operator-framework/operator-sdk/pull/5684/commits/2fd0a1e3d372482d9497e84787fb27bca30e883a

**Review by laxmikantbpandhare — APPROVED** [review]
/lgtm

**Pull request merged**